src/Microsoft.ML.StandardLearners/Standard/SdcaMultiClass.cs

```diff
@@ -29,14 +29,14 @@ namespace Microsoft.ML.Trainers
 {
     // SDCA linear multiclass trainer.
     /// <include file='doc.xml' path='doc/members/member[@name="SDCA"]/*' />
-    public class SdcaMultiClassTrainer : SdcaTrainerBase<SdcaMultiClassTrainer.Arguments, MulticlassPredictionTransformer<MulticlassLogisticRegressionModelParameters>, MulticlassLogisticRegressionModelParameters>
+    public class SdcaMultiClassTrainer : SdcaTrainerBase<SdcaMultiClassTrainer.Options, MulticlassPredictionTransformer<MulticlassLogisticRegressionModelParameters>, MulticlassLogisticRegressionModelParameters>
     {
         public const string LoadNameValue = "SDCAMC";
         public const string UserNameValue = "Fast Linear Multi-class Classification (SA-SDCA)";
         public const string ShortName = "sasdcamc";
         internal const string Summary = "The SDCA linear multi-class classification trainer.";
 
-        public sealed class Arguments : ArgumentsBase
+        public sealed class Options : ArgumentsBase
```

Comment:
>ArgumentsBase [](start = 38, length = 13)

Are we planning to rename this too?

Reply:
At this point the focus is to get the public APIs fixed.  

Rename of  internal  classes / variables is low priority at this point.  

---
In reply to: [248102673](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248102673) [](ancestors = 248102673)